xray/core/common.py

```diff
@@ -272,6 +272,31 @@ def groupby(self, group, squeeze=True):
             group = self[group]
         return self.groupby_cls(self, group, squeeze=squeeze)
 
+    def rolling(self, min_periods=None, center=False, **kwarg):
+        """Returns a Rolling object for performing moving window operations.
+
+        Parameters
+        ----------
+        min_periods : int, default None
+            Minimum number of observations in window required to have a value
+            (otherwise result is NA).
+        center : boolean, default False
+            Set the labels at the center of the window.
+        kwarg : dim=window
+            dim : str
```

Comment:
I like this docstring format -- does it render in a reasonable way in the API docs?